src/FINDOLOGIC/Export/Data/Property.php

```diff
@@ -88,7 +77,8 @@ protected function setValues($values)
          */
         array_walk($values, function ($item, $key) {
```

Comment:
Is it not possible to add typings to this closure?

Reply:
I added type hintings to the closures in other classes where possible. For this one, we may not add the type hint because we would skip our specific warning from the check, if the provided array is associative. 

I'll have to leave it like that for now.